packages/test-interface-criteria/src/criteria/TestOpportunityBookableOutsideValidFromBeforeStartDate.js

```diff
@@ -1,34 +1,40 @@
-const moment = require('moment');
-
-const { TestOpportunityBookable } = require('./TestOpportunityBookable');
-const { createCriteria } = require('./criteriaUtils');
+const { InternalCriteriaFutureScheduledOpportunity } = require('./internal/InternalCriteriaFutureScheduledOpportunity');
+const { getRemainingCapacity, createCriteria, mustBeWithinBookingWindow } = require('./criteriaUtils');
 
 /**
+ * @typedef {import('../types/Criteria').OpportunityConstraint} OpportunityConstraint
  * @typedef {import('../types/Criteria').OfferConstraint} OfferConstraint
  */
 
 /**
  * @type {OfferConstraint}
  */
-function outsideValidFromBeforeStartDate(offer, opportunity) {
-  return (Array.isArray(offer.availableChannel) && offer.availableChannel.includes('https://openactive.io/OpenBookingPrepayment'))
-    && offer.advanceBooking !== 'https://openactive.io/Unavailable'
-    && (offer.validFromBeforeStartDate && moment(opportunity.startDate).subtract(moment.duration(offer.validFromBeforeStartDate)).isAfter());
+function mustBeOutsideBookingWindow(offer, opportunity) {
+  return offer.validFromBeforeStartDate && !mustBeWithinBookingWindow(offer, opportunity);
 }
 
 /**
- * Implements https://openactive.io/test-interface#TestOpportunityBookableOutsideValidFromBeforeStartDate
```

Comment:
why has this doc been removed?

Reply:
It wasn't intentional - I'll restore it.

Reply:
Restored.